index.html

```diff
@@ -654,5 +654,327 @@ <h2>Exposing change of MediaStreamTrack configuration</h2>
     </p>
     </div>
   </section>
+  <section>
+    <h2>Human face segmentation</h2>
+    <p>Human face metadata describes the geometrical information of human faces
+    in video frames. It can be set by web applications using the standard means
+    when creating {{VideoFrameMetadata}} for {{VideoFrame}}s or it can be set by
+    a user agent when the media track constraint, defined below, is used to
+    enable face detection for the {{MediaStreamTrack}} which provides the
+    {{VideoFrame}}s.</p>
+    <p>The facial metadata can be used by video encoders to enhance the quality
+    of the faces in encoded video streams or for other suitable purposes.</p>
+    <section>
+      <h3>{{VideoFrameMetadata}}</h3>
+      <pre class="idl">
+partial dictionary VideoFrameMetadata {
+  sequence&lt;Segment&gt; segments;
+};</pre>
+      <section class="notoc">
+        <h4>Members</h4>
+        <dl class="dictionary-members" data-link-for="VideoFrameMetadata" data-dfn-for="VideoFrameMetadata">
+          <dt><dfn><code>segments</code></dfn> of type <code>sequence&lt;{{Segment}}&gt;</code></dt>
+          <dd>
+            <p>The set of known geometrical segments in the video frame.</p>
+          </dd>
+        </dl>
+      </section>
+    </section>
+    <section>
+      <h3>{{Segment}}</h3>
+      <pre class="idl">
+dictionary Segment {
+  required SegmentType type;
+  required long        id;
+  long                 partOf;
+  required float       probability;
+  Point2D              centerPoint;
+  DOMRectReadOnly      boundingBox;
```

Comment:
This uses what to me seems like a convenience interface inside a dictionary, which seems a bit unusual, and might be odd since VideoFrameMetadata might be used as input to the VideoFrame constructor.

Suggest using DOMRectInit instead.

Reply:
Looking at shape detection API, they went with DOMRectReadOnly.
I'll file an issue to see how we should handle the situation here, it seems we should converge somehow between the two APIs or at least know which model is the reference for future developments.